plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java

```diff
@@ -901,6 +929,55 @@ public void testTextualPredicatePushdown()
         assertThat(query("SELECT regionkey, nationkey, name FROM nation WHERE name = 'romania'"))
                 .returnsEmptyResult()
                 .isFullyPushedDown();
+        assertThat(query("SELECT regionkey, nationkey, name FROM nation WHERE name = 'romania' OR comment = 'P'"))
+                .returnsEmptyResult()
+                .isFullyPushedDown();
+
+        String withConnectorExpression = " OR some_column = 'x'";
+        try (TestTable table = new TestTable(
+                onRemoteDatabase(),
+                "tpch.test_textual_predicate_pushdown",
+                """
+                        (
+                        some_column String,
+                        a_string String,
+                        a_string_alias text,
```

Comment:
It worked, so even better. E.g. `string` does not work, so only `String` can be used. 
https://clickhouse.com/docs/en/sql-reference/data-types/string 
Here it's specified in the UPPERCASE.
It can be as it is.